### PR TITLE
Update Main references overview

### PR DESCRIPTION
**Pull request by shraddhavp**
this PR contains the references section overview.

### CODE REVIEW

File: docusaurus/docs/references/overview.mdx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-sidebar_label: 'References: Overview'
+title: WECHATY REFERENCES
+sidebar_label: 'References'
 ---
 
 import { Toc as ApiToc }  from '../api/overview.mdx'
 import { Toc as SpecToc } from '../specs/overview.mdx'
 
-References contain technical reference for APIs and other aspects of Wechaty's machinery. They describe how it works and how to use it but assume that you have a basic understanding of key concepts.
+The reference guide contains are a technical description of the machinery. They are code determined, information oriented. The references guides contain clear descriptions of Classes, API, Functions, Methods, Attributes, and many more technical aspects of the software. The Wechaty references aims to guide the end-users completely about Wechaty.
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/docs/references/overview.mdx]: aims -> aim ?

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

File: docusaurus/docs/references/overview.mdx (hunk 1 above)
Comment (shraddhavp) [reply docusaurus/docs/references/overview.mdx]: done @Rohitesh-Kumar-Jain  thanks

**Comment by shraddhavp** [comment]
@Soumi7

**Review by proudofsimin — APPROVED** [review]

File: docusaurus/docs/references/overview.mdx [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
 slug: /references/
-title: Wechaty references overview
-sidebar_label: 'References: Overview'
+title: WECHATY REFERENCES
```
Comment (lijiarui) [inline docusaurus/docs/references/overview.mdx]: I think `Wechaty References Overview` will be better.

File: docusaurus/docs/references/overview.mdx [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+sidebar_label: 'References'
```
Comment (lijiarui) [inline docusaurus/docs/references/overview.mdx]: I think we should change to `Overview` as the other part is written.

**Review by lijiarui — DISMISSED** [review]

**Review by Soumi7 — APPROVED** [review]

File: docusaurus/docs/references/overview.mdx (hunk 2 above)
Comment (shraddhavp) [reply docusaurus/docs/references/overview.mdx]: @lijiarui done

File: docusaurus/docs/references/overview.mdx (hunk 3 above)
Comment (shraddhavp) [reply docusaurus/docs/references/overview.mdx]: @lijiarui done

File: docusaurus/docs/references/overview.mdx [hunk 4]
```diff
@@ -1,13 +1,13 @@
 ---
 slug: /references/
-title: Wechaty references overview
-sidebar_label: 'References: Overview'
+title: WECHATY REFERENCES OVERVIEW
```
Comment (huan) [inline docusaurus/docs/references/overview.mdx:3]: I believe it is not a good title with all UPPER CAP?

Comment (shraddhavp) [reply docusaurus/docs/references/overview.mdx:3]: ok @huan

Comment (huan) [reply docusaurus/docs/references/overview.mdx:3]: You have read my comment and reply OK, but without changing it before merging it.

I think next time I will put a "Request changes" on your PR before everything is really good to go.

Please fix this problem by submitting another PR.

Comment (shraddhavp) [reply docusaurus/docs/references/overview.mdx:3]: @huan actually i had merged the PR before the comments from you,the next day I got the change request comment from you, will be doing the changes  by creating a new PR was busy this week so could not work on the issue. Will do it soon .

**Pull request merged**